content/applications/services/helpdesk/overview/receiving_tickets.rst

```diff
@@ -256,18 +200,83 @@ button which contains a link to join directly.
 Create a support ticket from a live chat session
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Once live chat is enabled, operators will be able to communicate with site visitors in real time.
+Once live chat is enabled, operators are able to communicate with site visitors in real time.
 
 During the conversation, an operator can use the shortcut :doc:`command
-</applications/websites/livechat/responses>` `/helpdesk` to create a ticket without leaving
-the chat window. The transcript from the conversation will be added to the new ticket, under the
+</applications/websites/livechat/responses>` `/ticket` to create a ticket without leaving the chat
+window. The transcript from the conversation is added to the new ticket, under the
 :guilabel:`Description` tab.
 
+.. _receiving_tickets/website-form:
+
+Website Form
+------------
+
+Enabling the *Website Form* setting adds a new page to the website with a customizable form. A new
+ticket is created once the required form fields are filled out and submitted.
+
+To activate the website form, navigate to a team's settings page under :menuselection:`Configuration
+--> Helpdesk Teams`. Find the :guilabel:`Website Form` feature under the :guilabel:`Help Center`
+section, and check the box.
+
+After the feature is activated, click the :guilabel:`Go to Website` smart button at the top of the
+that :guilabel:`Teams` settings page to view and edit the new website form, which is created
```

Comment:
```suggestion
After the feature is activated, click the :guilabel:`Go to Website` smart button at the top of the
:guilabel:`Teams` settings page to view and edit the new website form, which is created
```

Reply:
Also, I got the error when trying to click the Go To Website smartbutton: The visibility of the team needs to be set as "Invited portal users and all internal users" in order to use the website form." Not sure if this is standard, but might want to mention this?